scripts/commands/updateApiDocs.ts

```diff
@@ -38,7 +38,7 @@ interface Arguments {
   version: string;
 }
 
-type Pkg = {
+export type Pkg = {
```

Comment:
I'm surprised this works. It creates a circular dependency. Does the script run? 

Cleaner would be to move this type definition to `lib/`, e.g. `lib/updateApi.ts`.

Reply:
It does run, but I've moved it to `lib/sharedTypes.ts`